raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
Well yes and no. Our tests use two different blockchain backends. One is geth (which you used) and the other is tester. Both throw an exception, but a different one and in different places.

I am putting a comment in the code to signify where these exceptions can be raised.

In geth's case the `proxy.channelManagerByToken.call(token_address)` does indeed return `''`. But then an exception is thrown when this empty string is attempted to instantiate a ChannelManager.

Perhaps a TODO for the future would be to do some changes in the code there to fail earlier and detect an unregistered token earlier.